searches/jump_search.py

```diff
@@ -1,7 +1,28 @@
+"""
+Pure Python implementation of the jump search algorithm.
+This algorithm iterates through a sorted collection with a step of n^(1/2),
+until the element compared is bigger than the one searched.
+It will then perform a linear search until it matches the wanted number.
+If not found, it returns -1.
+"""
+
 import math
 
 
-def jump_search(arr, x):
+def jump_search(arr: list, x: int) -> int:
```

Comment:
```suggestion
def jump_search(arr: List[int], x: int) -> int:
```
First import List from typing

Reply:
This limits us to only the __int__ data type.  Without this specificity, would the algorithm work with strings, tulles, floats?